corehq/motech/generic_inbound/views.py

```diff
@@ -207,7 +207,7 @@ def _generic_inbound_api(api, request):
     except UserError as e:
         return JsonResponse({'error': str(e)}, status=400)
     except GenericInboundRequestFiltered:
-        return JsonResponse({}, status=204)
+        return HttpResponse(status=204)  # no body for 204 (RFC 7230)
```

Comment:
They are implemented but as long as we're returning a 204 (no content) then we should not include a body since that's what the HTTP spec indicates. If we do return a body it may even get removed by proxy servers in between the client and server.

The client can know that the request was not processed because of the 204 status code. 

We don't want to make it an error code otherwise the client may retry.